### DISCUSSION TITLE
generic Failed

### DISCUSSION THREAD

**Discussion by OpenDtu600** [tbnobody/OpenDTU#195] [General]
Hi habe folgendes Problem 
[.pio\build\generic\src\MqttPubilshing.cpp.o] Error1
[.pio\build\generic\src\MqttSettinhs.cpp.o] Error1
generic Failde

Bauteile: AZDelivery NRF24L01 und Mikrocontroller ESP-WROOM-32 Chip WiFi NodeMCU-32S ESP-WROOM-32.

Kann mir Jemand da behilflich sein?

VG

**Comment by tbnobody** [comment]
Veröffentliche doch bitte die komplette Fehlermeldung.

**Reply by OpenDtu600** [reply]
Hi hier das gesamte Fehlerbild. Hoffe sie können mir da weiterhelfen. Danke


![1](https://user-images.githubusercontent.com/115184697/194557867-9edb5da0-077f-42b9-89ec-b1862730a9e3.jpg)
![2](https://user-images.githubusercontent.com/115184697/194557884-e00d9314-09c6-4347-ba0b-d94d354a35e7.jpg)
![3](https://user-images.githubusercontent.com/115184697/194557895-4bd9669e-3556-41bb-9eb6-6ad5369a1426.jpg)

**Comment by tbnobody** [comment]
Es sieht aus als würde hier irgend etwas mit dem PlatformIO installation nicht passen. Die Fehlermeldung sagt ja das die stdint.h nicht gefunden werden kann. Diese Datei ist aber eine standard C Datei die bei PlatformIO dabei ist.

**Reply by OpenDtu600** [reply]
Auch nach einer neu installation habe ich das selbe problem.

**Comment by sivar2311** [comment]
- VS-Code beenden
- Das Verzeichnis `C:\Users\Ufunk Cetin\.platformio\packages\framework-arduinoespressif32` löschen. 
- VS-Code neu starten und das Projekt neu kompilieren

Das Framework wird durch PlatformIO automatisch neu installiert (kann etwas dauern)

**Reply by OpenDtu600** [reply]
Habe wie beschrieben die framework-arduinoespressif32 gelöscht und neu gestartet. Leider habe ich noch das selbe problem.
